packages/components/stories/AvroViewer.js

```diff
@@ -0,0 +1,605 @@
+			},
+			parking: { lot: false, valet: false, garage: false },
+			take_reservations: true,
+			noise_level: 'quiet',
+		},
+		location: {
+			zipType: 'STANDARD',
+			zip: 72132,
+			decommisioned: false,
+			taxReturnsFiled: 1400,
+			location: 'NA- US - AR - REDFIELD',
+			estimatedPopulation: 2653,
+			locationType: 'PRIMARY',
+			totalWages: 56190766,
+			state: 'AR',
+			longitude: -92.18,
+			latitude: 34.44,
+			city: 'REDFIELD',
+		},
+	},
+	{
+		business_id: 2,
+		name: "Nancy's Club",
+		category: 'Club',
+		rating: 2,
+		null_value: null,
+		num_of_reviews: 3779,
+		attributes: {
+			good_for: {
+				dessert: false,
+				kids: true,
+				drinks: false,
+				breakfast: false,
+				lunch: false,
+				dinner: true,
+			},
+			parking: { lot: true, valet: true, garage: false },
+			take_reservations: true,
+			noise_level: 'average',
+		},
+		location: {
+			zipType: 'PO BOX',
+			zip: 88221,
+			decommisioned: false,
+			taxReturnsFiled: 967,
+			location: 'NA-US - NM - CARLSBAD',
+			estimatedPopulation: 1638,
+			locationType: 'PRIMARY',
+			totalWages: 37060120,
+			state: 'NM',
+			longitude: -104.23,
+			latitude: 32.4,
+			city: 'CARLSBAD',
+		},
+	},
+	{
+		business_id: 3,
+		name: "Cecelia's Club",
+		category: 'Cafe',
+		rating: 4,
+		null_value: null,
+		num_of_reviews: 16547,
+		attributes: {
+			good_for: {
+				dessert: true,
+				kids: false,
+				drinks: false,
+				breakfast: true,
+				lunch: false,
+				dinner: false,
+			},
+			parking: { lot: true, valet: true, garage: false },
+			take_reservations: false,
+			noise_level: 'noisy',
+		},
+		location: {
+			zipType: 'PO BOX',
+			zip: 47445,
+			decommisioned: false,
+			taxReturnsFiled: 123,
+			location: 'NA-US - IN - MIDLAND',
+			estimatedPopulation: 123,
+			locationType: 'PRIMARY',
+			totalWages: 456,
+			state: 'IN',
+			longitude: -87.16,
+			latitude: 39.09,
+			city: 'MIDLAND',
+		},
+	},
+	{
+		business_id: 4,
+		name: "Gordon's Bar",
+		category: 'Cafe',
+		rating: 1,
+		null_value: null,
+		num_of_reviews: 152,
+		attributes: {
+			good_for: {
+				dessert: false,
+				kids: false,
+				drinks: true,
+				breakfast: true,
+				lunch: true,
+				dinner: true,
+			},
+			parking: { lot: true, valet: false, garage: true },
+			take_reservations: true,
+			noise_level: 'noisy',
+		},
+		location: {
+			zipType: 'STANDARD',
+			zip: 65638,
+			decommisioned: false,
+			taxReturnsFiled: 123,
+			location: 'NA- US - MO - DRURY',
+			estimatedPopulation: 123,
+			locationType: 'PRIMARY',
+			totalWages: 456,
+			state: 'MO',
+			longitude: -92.32,
+			latitude: 36.92,
+			city: 'DRURY',
+		},
+	},
+];
+
+const schema = [
+	{
+		name: 'id',
+		doc: 'Id',
+		type: {
+			type: 'integer',
+		},
+		'@talend-quality@': {
+			0: 5,
+			1: 65,
+			'-1': 30,
+		},
+	},
+	{
+		name: 'name',
+		doc: 'Name',
+		type: {
+			type: 'string',
+			dqType: 'Recipe',
+			dqTypeKey: 'RECIPE',
+		},
+		'@talend-quality@': {
+			0: 5,
+			1: 65,
+			'-1': 30,
+		},
```

Comment:
Wait a minute. :thinking: 
This isn't an Avro Schema. 
It looks like a weird kind of wrapped, hand-made JSON.